src/util/services_utils.js

```diff
@@ -0,0 +1,23 @@
+/* Copyright (C) 2024 NooBaa */
+'use strict';
+
+const signature_utils = require('./signature_utils');
+const { RpcError } = require('../rpc');
+
+const sdk_type_enum = {
+    OBJECT_SDK: 'object_sdk',
+    STS_SDK: 'sts_sdk',
+    ACCOUNT_SDK: 'account_sdk',
+};
+
+function authorize_request_account_by_token(token, requesting_account) {
```

Comment:
I think it's better to move this function to signature_utils or endpoint_utils

Reply:
I moved it to `signature_utils` ( I didn't choose `endpoint_utils` is not under `src/util` but `src/endpoint` and its current functions are only used by `endpoint.js`).